wpilibc/src/main/python/semiwrap/simulation/DriverStationSim.yml

```diff
@@ -1,6 +1,22 @@
 classes:
   wpi::sim::OpModeOptions:
-    ignore: true
+    ignored_bases:
+    - std::span<HAL_OpModeOption>
+    force_no_trampoline: true
+    methods:
+      OpModeOptions:
+        overloads:
+          "":
+          HAL_OpModeOption*, int32_t:
+    inline_code: |
+      .def("__len__", [](const OpModeOptions &self) { return self.size(); })
+      .def("__getitem__", [](const OpModeOptions &self, int index) {
+        return self[index];
+      })
+      .def("__iter__", [](OpModeOptions * self) {
```

Comment:
I prefer to use `&self` instead of `* self` because pybind11 is stupid and will let you pass None as self if it's a pointer so it'll segfault, but it'll only be a python error if it's a reference.

Try it. It's silly. Nanobind changes this default I think though.